portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/plus.vm

```diff
@@ -0,0 +1,360 @@
+##<!-- START VM plus.vm --> ## Comments seen before doctype. Internet Explorer will go into the quirks mode.
```

Comment:
maybe need to discuss this

Reply:
This is a copy of the file site.vm in the same folder with as few changes as possible to support plus.   The complaint you have is also present in site.vm and I won't fix it in plus.vm without also fixing it identically in site.vm.  And frankly, when trinity comes in - both these files will get a lot of eyes and review.  I need to keep site.vm and plus.vm in parallel so that when trinity makes major changes to site.vm - I can get plus.vm working again post-trinity.